libraries/scrcpy/src/control/serializer.ts

```diff
@@ -88,7 +87,7 @@ export class ScrcpyControlMessageSerializer {
      */
     public injectTouch(message: Omit<ScrcpyInjectTouchControlMessage, "type">) {
```

Comment:
Make these functions returns serialized messages.

Add a new wrapper class for writing to a WritableStream